docs/resources/azure_active_directory_domain_services.md

```diff
@@ -0,0 +1,101 @@
+  #...
+end
+```
+## Parameters
+
+The following parameters can be passed for targeting specific Domains.
+
+| Name              | Description                                                 | Example                             |
+|-------------------|-------------------------------------------------------------|-------------------------------------|
+| filter            | A hash containing the filtering options and their values. The `starts_with_` operator can be used for fuzzy string matching. Parameter names are in snake_case. | `{ starts_with_given_name: 'J', starts_with_department: 'Core', country: 'United Kingdom', given_name: John}` |
+| filter_free_text  | [OData](https://www.odata.org/getting-started/basic-tutorial/) query string in double quotes, `"`. Property names are in camelcase, refer to [here](https://docs.microsoft.com/en-us/graph/query-parameters#filter-parameter) for more information. | `"startswith(displayName,'J') and surname eq 'Doe'"` or `"userType eq 'Guest'"` |
+
+It is advised to use these parameters to narrow down the targeted resources at the server side, Azure Graph API, for a more efficient test.
+
+## Properties
+
+| Property              | Description                                           | Filter Criteria<superscript>*</superscript>  |
+|-----------------------|-------------------------------------------------------|----------------------------------------------|
+| ids                   | A list of fully qualified name of the domain.         | `id`                                         |
+| authentication_types  | A list of the configured authentication types for the domain.| `authentication_type`                 |
+| availability_statuses | A list of domain entities when verify action is set.  | `availability_status`                        |
+| is_admin_managed      | A list of admin managed configuration.                | `is_admin_managed`                           |
+| is_default            | A list of flags to indicate if it is default domains. | `is_default`                                 |
+| is_initial            | A list of flags to indicate if initial domains created by Microsoft Online Services.| `is_initial`   |
+| is_root               | A list of flags to indicate if verified root domain.  | `is_root`                                    |
+| is_verified           | A list of flags to indicate if the domain has completed domain ownership verification.| `is_verified`|
+| passwordNotificationWindowInDays | A list of password notification window days.| `passwordNotificationWindowInDays`          |
+| passwordValidityPeriodInDays | A list of password validity period in days.    |  `passwordValidityPeriodInDays`              |
+| supportedServices     | A list of capabilities assigned to the domain.        |  `supportedServices`                         |
+| state                 | A list of asynchronous operations scheduled.          |  `state`                                     |
```

Comment:
Why state is not pluralized?

Reply:
table schema that was auto-generated based on the response for domain services .. 
<img width="975" alt="Screenshot 2021-07-29 at 7 02 55 PM" src="https://user-images.githubusercontent.com/80091550/127501183-94712e19-8757-46c9-8cd1-13512d3a53bd.png">